ui/static/ui/js/learning_resources.jsx

```diff
@@ -379,12 +386,17 @@ define('learning_resources',
         thiz.setState({
           message: {error: "Unable to save form"}
         });
+      }).always(function() {
+        thiz.setState({loaded: true});
```

Comment:
No agreed, it wouldn't trigger anything in and of itself. What I meant is, the "loaded" state being set to true implies that the tab content was loaded successfully. If we ever relied on that state for anything else, like allowing XML content to be edited for example, or letting applied terms be modified, we might assume the content was successfully loaded, when it's not necessarily; that state is set to true even on failure, to remove the spinner. Really just a semantic thing, I think. Does that make sense?